rampwf/utils/importing.py

```diff
@@ -3,9 +3,10 @@
 
 """
 import importlib
+from .sanitize import _sanitize_input
 
 
-def import_module_from_source(source, name):
+def import_module_from_source(source, name, sanitize=False):
```

Comment:
Can't make `True` the default, as it's also used in various notebooks to load `problem.py`